cert/openssl.cnf

```diff
@@ -129,15 +129,16 @@ basicConstraints 				= critical, CA:true
 keyUsage 						= critical, digitalSignature, cRLSign, keyCertSign
 
 ####################################################################
-#Not used yet
+
 [ crl_section ]
 
-URI.0 = http://crl3.digicert.com/WInnForumCA.crl
-URI.1 = http://crl4.digicert.com/WInnForumCA.crl
+#URI.0 = http://crl3.digicert.com/WInnForumCA.crl
```

Comment:
Please remove commented code.

Reply:
The   changes to this file are no longer required as the required changes have been   merged as part CRL server PR#469